source/utils/of-type-with-name.ts

```diff
@@ -0,0 +1,22 @@
+import test from '../test';
+import {BasePredicate} from '../predicates/base-predicate';
+
+/**
+Test all the values in the collection against a provided predicate.
+
+@hidden
+@param source Source collection to test.
+@param name The name to call the collection of values, such as "values", "keys".
```

Comment:
I don't think this needs to be a separate assertion. You can just add an optional parameter to `of-type`.

Reply:
I updated the PR. 
After we migrate all usages of `ofType` to specify the optional `name` parameter, we can change that to be required.

Reply:
> After we migrate all usages of ofType to specify the optional name parameter, we can change that to be required.

Can you add a code comment about that? So we don't forget.

Reply:
Sure, updated the comment.